### ISSUE TITLE
OpenSSL::ASN1::Sequence#each is broken

### ISSUE THREAD

**Issue by cielavenir** [ruby/openssl#115]
Ruby 2.0.0

```
irb> OpenSSL::ASN1::Sequence.new([OpenSSL::ASN1::Integer(1)]).each{|e|p e}
#<OpenSSL::ASN1::Integer:0x007f997dafffa0 @tag=2, @value=1, @tagging=nil, @tag_class=:UNIVERSAL, @infinite_length=false>
=> #<OpenSSL::ASN1::Sequence:0x007f997dafff50 @tag=16, @value=[#<OpenSSL::ASN1::Integer:0x007f997dafffa0 @tag=2, @value=1, @tagging=nil, @tag_class=:UNIVERSAL, @infinite_length=false>], @tagging=nil, @tag_class=:UNIVERSAL, @infinite_length=false>
```

Ruby 2.4.0

```
irb> OpenSSL::ASN1::Sequence.new([OpenSSL::ASN1::Integer(1)]).each{|e|p e}
=> #<OpenSSL::ASN1::Sequence:0x007fa5bf044d88 @tag=16, @value=[#<OpenSSL::ASN1::Integer:0x007fa5bf044e28 @tag=2, @value=1, @tagging=nil, @tag_class=:UNIVERSAL, @infinite_length=false>], @tagging=nil, @tag_class=:UNIVERSAL, @infinite_length=false>
```

It is obvious that sequence is not enumerated in Ruby 2.4. I believe that this is coming from `ossl_asn1cons_each`, which was changed between OpenSSL 2.0.0beta1 and 2.0.0.
I have confirmed that my ASN1 manipulation is all right in Ruby 1.9-2.3.0  using travis.

**Comment by rhenium** [comment]
This should have been fixed by #96. Try openssl gem version 2.0.3.

**Comment by cielavenir** [comment]
Ah, thank you so much. Perhaps for travis we have to wait for 2.4.1...
